docs/source/_static/js/custom.js

```diff
@@ -1,3 +1,26 @@
+// These two things need to be updated at each release for the version selector.
+// Last stable version
+let stableVersion = "v2.11.0"
+// Dictionary doc folder to label
+let versionMapping = {
```

Comment:
We don't expect that to change, so it should be `const`

```suggestion
const stableVersion = "v2.11.0"
// Dictionary doc folder to label
const versionMapping = {
```